### DISCUSSION TITLE
More donation options (cryptocurrency).

### DISCUSSION THREAD

**Discussion by canisllup** [dyc3/steamguard-cli#225] [General]
Please consider adding donation addresses for cryptocurrency. Preferably, Nano/Monero. Thanks for the great app!

**Comment by dyc3** [comment]
This is pretty unlikely, though I do appreciate the sentiment. I already don't receive much in the first place, so managing a bunch of crypto wallets seems like too much hassle for not much reward.

**Comment by canisllup** [comment]
Understandable, thanks for the timely response!
